diff --git a/node_modules/react-native/Libraries/Components/View/ViewPropTypes.d.ts b/node_modules/react-native/Libraries/Components/View/ViewPropTypes.d.ts
index 53f5e82..8c67a32 100644
--- a/node_modules/react-native/Libraries/Components/View/ViewPropTypes.d.ts
+++ b/node_modules/react-native/Libraries/Components/View/ViewPropTypes.d.ts
@@ -184,7 +184,7 @@ export interface ViewProps
    * the Z-index of sibling views always takes precedence if a touch
    * hits two overlapping views.
    */
-  hitSlop?: Insets | undefined;
+  hitSlop?: null | Insets | number | undefined;
 
   /**
    * Used to reference react managed views from native code.
